agent/engine/docker_image_manager_integ_test.go

```diff
@@ -530,24 +524,20 @@ func TestImageWithSameIDAndDifferentNames(t *testing.T) {
 
 	// Verify images are removed by docker
 	_, err = taskEngine.(*DockerTaskEngine).client.InspectImage(imageID1)
-	assert.Equal(t, docker.ErrNoSuchImage, err, "Image was not removed successfully")
+	assert.True(t, client.IsErrNotFound(err), "Image was not removed successfully")
 }
 
-// renameImage retag the image and delete the original tag
-func renameImage(original, repo, tag string, client *docker.Client) error {
-	err := client.TagImage(original, docker.TagImageOptions{
-		Repo:  repo,
-		Tag:   tag,
-		Force: false,
-	})
+// renameImage retag the image with the target tag and delete the source tag
+func renameImage(ctx context.Context, source string, target string, client *client.Client) error {
```

Comment:
Do we really need to pass in `ctx` to `renameImage`? Can't we just use `context.TODO()` in `client.ImageTag`